medusa/providers/torrent/json/t411.py

```diff
@@ -92,20 +188,38 @@ def search(self, search_strings, age=0, ep_obj=None):
 
             for search_string in search_strings[mode]:
                 if mode != 'RSS':
-                    logger.log('Search string: {search}'.format
-                               (search=search_string), logger.DEBUG)
+                    season = episode = None
+                    if isinstance(search_string, tuple):
```

Comment:
did you see entire code?
Or its a list or tuple. No other alternative
Why check collections?!

You are talking theory. General code. 
Tuple is only for t411. All others use lists in search strings